action.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Install Shorebird'
+description: 'Installs shorebird'
+runs:
+  using: "composite"
+  steps:
+    - id: install
```

Comment:
Since shorebird requires Flutter and Dart should we also install those as steps?

Reply:
Good idea, will add

Reply:
Done.